apps/SageAccountWeb/src/components/RegisterAccount1.tsx

```diff
@@ -173,31 +180,38 @@ export const RegisterAccount1 = () => {
                     <SourceAppLogo />
                   </div>
                   {page === Pages.CHOOSE_REGISTRATION && (
-                    <div style={{ marginTop: '30px' }}>
-                      <Button
-                        onClick={() => setPage(Pages.GOOGLE_REGISTRATION)}
-                        sx={chooseButtonSx}
-                        variant="outlined"
-                      >
-                        <img
-                          className="googleLogo"
-                          src={GoogleLogo}
-                          alt="Google Logo"
-                          style={{ width: 25, marginRight: 5 }}
-                        />
-                        <span className="signInText">
-                          Create account with Google
-                        </span>
-                      </Button>
-                      <Button
-                        onClick={() => setPage(Pages.EMAIL_REGISTRATION)}
-                        sx={chooseButtonSx}
-                        variant="outlined"
-                      >
-                        <IconSvg icon="email" sx={{ marginRight: '5px' }} />
-                        Create account with your email
-                      </Button>
-                    </div>
+                    <>
+                      <div style={{ marginTop: '30px' }}>
+                        <Button
+                          onClick={() => setPage(Pages.GOOGLE_REGISTRATION)}
+                          sx={chooseButtonSx}
+                          variant="outlined"
+                        >
+                          <img
+                            className="googleLogo"
+                            src={GoogleLogo}
+                            alt="Google Logo"
+                            style={{ width: 25, marginRight: 5 }}
+                          />
+                          <span className="signInText">
+                            Create account with Google
+                          </span>
+                        </Button>
+                        <Button
+                          onClick={() => setPage(Pages.EMAIL_REGISTRATION)}
+                          sx={chooseButtonSx}
+                          variant="outlined"
+                        >
+                          <IconSvg icon="email" sx={{ marginRight: '5px' }} />
+                          Create account with your email
+                        </Button>
+                      </div>
+                      {lastLoginInfo && (
+                        <Box position="absolute" bottom={0} pb="54px" pr="64px">
+                          {lastLoginInfo}
+                        </Box>
+                      )}
```

Comment:
Display the notice box if the LastLoginInfo component has something to display, i.e. it isn't null